lib/github.atd

```diff
@@ -106,6 +119,27 @@ type issue = {
   ~comments <ocaml default="0">: int;
 }
 
+type compare = {
+  url: string;
+  html_url: string;
+  permalink_url: string;
+  diff_url: string;
+  patch_url: string;
+  base_commit: api_commit;
+  merge_base_commit: api_commit;
+  status: string;
+  ahead_by: int;
+  behind_by: int;
+  total_commits: int;
+  commits: api_commit list;
+  files: file list;
```

Comment:
```suggestion
  ~files <ocaml default="[]">: file list;
```

`files` is apparently not a required field (see "Response schema" in [doc](https://docs.github.com/en/rest/commits/commits#compare-two-commits))